packages/rich-text-editor/src/RichTextEditor/RichTextEditor.playwright.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import { expect, test } from "@playwright/test"
+
+const iframePath = "iframe.html?args=&id="
+const storyId = "components-rich-text-editor-rich-text-editor--default"
+
+test.beforeEach(async ({ page }) => {
+  // Go to the starting url before each test.
+  await page.goto(`./${iframePath}${storyId}`)
+})
+
+test.describe("Test list keyboard shortcuts", async () => {
+  test("<ul> can be created", async ({ page }) => {
+    const editor = await page.locator('div[role="textbox"]')
+
+    await editor.click()
+    await editor.type("-")
+    await editor.press("Space")
+    await editor.type("By the pricking of my thumbs")
+    await editor.press("Enter")
+    await editor.type("Something wicked this way comes")
+
+    expect(await editor.locator("ul li").count()).toBe(2)
```

Comment:
Again, super nitpicky, but technically `expect(await editor.locator("ul li").count()).toBe(2)` could have been `true` even before you interacted with the editor.
If you want to be super thorough you'd start the test with:
`expect(await editor.locator("ul li").count()).toBe(0)`

Reply:
Good tip, will amend that test